### PR TITLE
feat(user bulk upload): bulk upload ownidp

### PR DESCRIPTION
**Pull request by nidhigarg-bmw**
## Description

User bulk upload ownidp

## Why

Add functionality for ownidp scenario

## Issue

n/a

## Checklist

Please delete options that are not relevant.

- [X] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/portal-assets/blob/main/developer/Technical%20Documentation/Dev%20Process/How%20to%20contribute.md#commit-and-pr-guidelines)
- [X] I have performed a self-review of my own code
- [X] I have successfully tested my changes locally

### CODE REVIEW

**Review by manojava-gk — APPROVED** [review]
lftm

File: src/components/overlays/AddMultipleUser/index.tsx [hunk 1]
```diff
@@ -383,58 +396,85 @@ export default function AddMultipleUser() {
...
           </Button>
-        )}
-      </DialogActions>
+          {loading ? (
+            <LoadingButton
+              color="primary"
+              helperText=""
+              helperTextColor="success"
+              label=""
+              loadIndicator="Loading ..."
```
Comment (oyo) [inline src/components/overlays/AddMultipleUser/index.tsx:432]: Translation?

File: src/components/overlays/AddMultipleUser/index.tsx [hunk 2]
```diff
@@ -383,58 +396,85 @@ export default function AddMultipleUser() {
...
-      />
+  const renderMultiuserMainContent = () => {
+    return idps.length === 1 ? (
+      <>
+        <DialogHeader
+          {...{
+            title: t('content.usermanagement.addMultipleUsers.heading'),
+            intro: t('content.usermanagement.addMultipleUsers.intro'),
+            closeWithIcon: true,
+            onCloseWithIcon: () => dispatch(show(OVERLAYS.NONE, '')),
```
Comment (oyo) [inline src/components/overlays/AddMultipleUser/index.tsx:411]: use `dispatch(closeOverlay())`

**Review by oyo — CHANGES_REQUESTED** [review]

**Comment by nidhigarg-bmw** [comment]
@oyo  Made the changes

**Review by oyo — APPROVED** [review]

**Pull request merged**